pkgs/desktops/gnome-3/core/gnome-tour/default.nix

```
@@ -0,0 +1,79 @@
+{ stdenv
+, rustPlatform
+, gettext
+, meson
+, ninja
+, fetchFromGitLab
+, pkgconfig
+, gtk3
+, glib
+, gdk-pixbuf
+, desktop-file-utils
+, appstream-glib
+, wrapGAppsHook
+, python3
+, gnome3
+, config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "gnome-tour-unstable";
+  version = "2020-02-15";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = "gnome-tour";
+    rev = "129ec5d20a512ed4937da5088170c8ba61579c7b";
+    sha256 = "03if1y607jhy70rgb6bdicqj16c7khaa4wmiga7jrbbayf64nlgy";
+  };
+
+  cargoSha256 = "1gh2xvaq7v0zh375f98pbhjx65b07km47n2g5lplrxvznaw03c2a";
+
+  mesonFlags = [
+    "-Ddistro_name=NixOS"
+    "-Ddistro_icon_name=nix-snowflake"
+    "-Ddistro_version=20.03"
+  ];
+
+  nativeBuildInputs = [
+    pkgconfig
+    desktop-file-utils
+    meson
+    ninja
+    wrapGAppsHook
+    python3
+  ];
+
+  buildInputs = [
+    gtk3
+    glib
+    gdk-pixbuf
+    gettext
+  ];
+
+  # Don't use buildRustPackage phases, only use it for rust deps setup
+  configurePhase = null;
+  buildPhase = null;
+  checkPhase = null;
+  installPhase = null;
+
+  postPatch = ''
+    chmod +x build-aux/meson_post_install.py
+    patchShebangs build-aux/meson_post_install.py
+  '';
+
+  # passthru = {
+  #   updateScript = gnome3.updateScript {
+  #     packageName = pname;
+  #   };
+  # };
+
+  meta = with stdenv.lib; {
+    homepage = "https://gitlab.gnome.org/GNOME/gnome-tour";
+    description = "GNOME Greeter & Tour";
+    maintainers = gnome3.maintainers;
+    license = licenses.gpl3;
```

https://gitlab.gnome.org/GNOME/gnome-tour/blob/5dc5ec9d47ab739ff1f2a5c1a732aa7e5483e400/data/org.gnome.Tour.metainfo.xml.in.in#L6

```suggestion
    license = licenses.gpl3Plus;
```